ikenna-portfolio/portfolio/src/main/webapp/test.html

```diff
@@ -0,0 +1,267 @@
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>                         
+        </button>
+        <a class="navbar-brand" href="#">'kenna</a>
+        </div>
+        <div class="collapse navbar-collapse" id="myNavbar">
+          <ul class="nav navbar-nav navbar-right">
+            <li><a href="#">INTRO</a></li>
+            <li><a href="#">STEP</a></li>
+            <li><a href="#">PROJECTS</a></li>
+            <li><a href="#">CALISTHENICS</a></li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+
+    <!-- First Container -->
+    <div class="container-fluid bg-1 text-center">
+      <h3 class="margin">Ikenna's Portfolio</h3>
+      <img class="image-format" src="/images/img_baby.jpeg" alt="Youngin 'kenna">
+      <h3>I'm a Googler!!</h3>
+      <p class="desc">My name is <b>Ikenna Jonathan Elue</b>. I am a male identifying Nigerian American 
+      from Chicago who currently attends Northwestern University. In the past, I've worked with Chicago 
+      based industries such as JCC Apachi Camp (Jewish Community Center) and ASM (After School Matters). 
+      My favorite food is Nigerian jollof rice. I enjoy doing calisthenic based exercises, combination 
+      jump roping, and dancing. One non physical activity I enjoy is watching comedy animes or cartoons.
+      </p>
+      <br>
+      <p class="desc">Here's a link to my LinkedIn <a href="https://www.linkedin.com/in/ikenna-elue-a28798166/">page</a></p>
+      <p class="desc">Here's a link to my GitHub <a href="https://github.com/ike-elue">page</a></p>
+    </div>
+
+    <!-- Second Container -->
+    <div class="container-fluid bg-2 text-center">
+      <h3 class="margin">STEP So Far</h3>
+      <h3><u>TODO</u></h3>
+      <p>Click here to read a random quote:</p>
+      <button onclick="addRandomQuote()">Try It!</button>
+      <div id="quote-container"></div>
+    </div>
+
+    <!-- Third Container -->
+    <div class="container-fluid bg-3 text-center">
+      <h3 class="margin">Cool Projects</h3>
+      <div class="row">
+        <div class="column">
+          <button onclick="changeCycleGAN()">CycleGAN</button>
+        </div>
+        <div class="column">
+          <button onclick="changeDeepPhoto()">Deep Photo Style Transfer</button>
+        </div>
+        <div class="column">
+          <button onclick="changeMsgSys()">Message System</button>
+        </div>
+      </div>
+      <hr>
+      <br>
+      <div class="row">
+        <div class="column">
+          <img id="pro-image1" class="image-format" src="" alt="">
+        </div>
+        <div class="column">
+          <img id="pro-image2" class="image-format" src="" alt="">
+        </div>
+        <div class="column">
+          <img id="pro-image3" class="image-format" src="" alt="">
+        </div>
+      </div>
+      <br>
+      <div class="row">
+        <div class="col-sm-8">
```

Comment:
Do you need to define this class and col-sm-4? It doesn't seem like they're being used in css.

Reply:
"col-sm" is defined by Bootstrap to have in ratio columns. A problem I found while trying to use this was that the columns were touching which I didn't want them to so I reused the "column definition" I had previously.